devlab/config.ini

```diff
@@ -11,6 +11,10 @@ grizzly_compute_ip = 192.168.1.5
 icehouse_compute_ip = 192.168.1.6
 juno_ip = 192.168.1.8
 
+http_proxy =
+https_proxy =
+ftp_proxy =
```

Comment:
This doesn't look right, if you want to leave these options available in config.ini – please comment those out and leave comments to the target audience, don't leave options unassigned.